ansible/roles/baselayout-windows/tasks/main.yml

```diff
@@ -17,10 +25,14 @@
       params: '/GitAndUnixToolsOnPath'
   - name: enable long paths in Git
     win_command: 'git config --global core.longpaths true'
+  - name: set Git user name
+    win_command: 'git config --global user.name "{{inventory_hostname}}"'
+  - name: set Git user email
+    win_command: 'git config --global user.email "ci@iojs.org"'
 
 # Necessary for junit output in Jenkins jobs
 - name: install tap2junit
-  win_command: 'pip2 install tap2junit'
+  win_command: 'pip3 install --upgrade tap2junit'
```

Comment:
I think on others we've done both pip2 and pip3 to make sure it's available on both versions of python, might that be necessary here too?

Reply:
Isn't that redundant? tap2junit is invoked by the build scripts, so we have control over what exactly is called. If we have both, that would either just ignore one (the one not first in the path) or call an arbitrary one depending on which one is first in the path. Since tap2junit supports Python 3, I figured we could just go with that.

@cclauss do you know if we need both?

Reply:
In this case we just need a single instance of tap2junit and I would prefer that Python 3 is the way that we install it.

Reply:
@rvagg I plan to land this as is, let me know if I missed something.